### DISCUSSION TITLE
olá pessoal acabei de conhecer o saite e achei foda d+.

### DISCUSSION THREAD

**Discussion by YugaVieira** [YuriAlessandro/gerador-ficha-tormenta20#365] [Geral]
Fala pessoal.
Meu o saite de você é incrivél !!!!!
mas não consigo exportar as fichas para o foundryVTT de forma funcinal.

**Comment by YuriAlessandro** [comment]
acho que essa função está mesmo deixando a desejar. Vou dar uma olhada melhor depois
